File: samples/client/petstore/csharp/SwaggerClientTest/Lib/SwaggerClient/src/main/csharp/IO/Swagger/Model/EnumClass.cs
using System;
using System.Linq;
using System.IO;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Runtime.Serialization;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace IO.Swagger.Model
{
    /// <summary>
    /// Gets or Sets EnumClass
    /// </summary>
    [JsonConverter(typeof(StringEnumConverter))]
    public enum EnumClass
    {
        
        /// <summary>
        /// Enum Abc for "_abc"
        /// </summary>
        [EnumMember(Value = "_abc")]
        Abc,
        
        /// <summary>
        /// Enum efg for "-efg"
        /// </summary>
        [EnumMember(Value = "-efg")]
        efg,
        
        /// <summary>
        /// Enum xyz for "(xyz)"
        /// </summary>
        [EnumMember(Value = "(xyz)")]
        xyz
    }

}

File: samples/client/petstore/java/okhttp-gson/src/main/java/io/swagger/client/model/EnumClass.java
package io.swagger.client.model;

import java.util.Objects;

import com.google.gson.annotations.SerializedName;


/**
 * Gets or Sets EnumClass
 */
public enum EnumClass {
  @SerializedName("_abc")
  _ABC("_abc"),

  @SerializedName("-efg")
  _EFG("-efg"),

  @SerializedName("(xyz)")
  _XYZ_("(xyz)");

  private String value;

  EnumClass(String value) {
    this.value = value;
  }

  @Override
  public String toString() {
    return String.valueOf(value);
  }
}

